lil_bank/accounts/forms.py

```diff
@@ -19,13 +21,39 @@ class SignUpForm(forms.Form):
     """
     first_name = forms.CharField(max_length=32)
     last_name = forms.CharField(max_length=32)
+    last_name = forms.CharField(max_length=32)
```

Comment:
It seems as though there are two declarations for the `last_name` variable, could you check on that?

Reply:
Resolved.